src/components/ListGrid.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="grid-table">                
+    <hr>
+
+    <div id="draw-table">
+		
+  	</div>
+		<hr>
+  </div>
+</template>
+
+<script>
+import EventBus from '../services/bus'
+
+export default {
+    name: 'ListGrid',
+    data () {
+        return {
+            grids: [],
+            numColumns: '',
+            numRows: ''
+        }
+    },
+    created: function() {
+        EventBus.$on('add-grid', (fil, row) => {
+            this.numColumns = fil;
+            this.numRows = row;                        
+            var codigoHTML="<table border=\"1\"><tbody>";
+            for (var i = 0; i < this.numRows; i++) {
+            codigoHTML+="<tr>";
+            for (var j = 0; j < this.numColumns; j++) {
+            codigoHTML+="<td WIDTH=\"55px\" HEIGHT=\"55px\"></td>";
+            }
+            codigoHTML+="</tr>";
+            }
+            codigoHTML+="</tbody></table>";
+            document.getElementById("draw-table").innerHTML = codigoHTML;
```

Comment:
use vuejs features instead of manual DOM manipulation, for example:
```html
<table>
  <tr v-for="row in rows">
    <td v-for="column in columns"></td>
  </tr>
</table>
```

Reply:
unreadable code, fix indentations